node-installer/runtimehandler/runtimehandler.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package runtimehandler
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/edgelesssys/contrast/node-installer/internal/constants"
+	"github.com/edgelesssys/contrast/node-installer/platforms"
+)
+
+// Name returns the name of the runtime handler for the given platform.
+func Name(platform platforms.Platform) (string, error) {
+	platformName := strings.ToLower(platform.String())
+
+	if strings.EqualFold(platformName, platforms.Unknown.String()) {
+		return "", fmt.Errorf("unsupported platform %s", platform)
+	}
+
+	// Replace dots to ensure a readable directory name used by the node-installer.
+	version := strings.ReplaceAll(constants.Version, ".", "-")
+
+	return fmt.Sprintf("contrast-cc-%s-%s", version, platformName), nil
```

Comment:
Just so I understand it correctly: the prefix up to excluding the platformName is unique per build? If the middle `%s` depends on the right `%s`, I'd prefer to swap them.